Automattic/wp-calypso
# Social Previews: Refactor tests to RTL

#### Proposed Changes

This PR refactors the social previews component tests to use `@testing-library/react`. We're also improving the assertions to use `jest-dom` ones where that makes sense.

There are some rough edges here and there, but this is just a first step towards improving those tests.

#### Testing Instructions

Verify tests still pass: `yarn run test-packages packages/social-previews/test/index.js`

Related to #70873
